openmc/element.py

```diff
@@ -181,6 +236,70 @@ def expand(self, percent, percent_type, enrichment=None,
             for nuclide in abundances.keys():
                 abundances[nuclide] /= sum_abundances
 
+        # Modify mole fractions if enrichment provided
+        # New treatment for arbitrary element
+        # Interpret required enrichment as weight
+        elif enrichment is not None and enrichment_target is not None:
+
+            # Check if is a single isotope mixture
+            if len(abundances) != 2:
+                msg = 'Element {0} does not consist of 2 isotopes. Thus it '\
+                      'cannot be enriched with in-build procedure. Please '\
+                      'enter isotopic abundances manually. '.format(self)
```

Comment:
Suggest minor rewording:
`             msg = 'Element {0} does not consist of 2 naturally-occurring isotopes. '\
                           'Therefore it cannot be enriched with the in-built procedure. Please '\
                       'enter isotopic abundances manually. '.format(self)`